src/app/modules/audience/views/engagement/highlights/highlights.component.html

```diff
@@ -2,5 +2,74 @@
   <div class="kFilters">
     <span class="kTitle">{{'app.engagement.highlights' | translate}}</span>
   </div>
+
+  <k-area-blocker [message]="_blockerMessage" [showLoader]="_isBusy">
+
+    <div class="kReportContent">
+      <app-report-tabs [tabs]="_tabsData"
+                       [compareMode]="_isCompareMode"
+                       (tabChange)="_onTabChange($event)"></app-report-tabs>
+      <div class="kDimensions">
+        <app-time-units [dateFilter]="dateFilterComponent"></app-time-units>
+      </div>
+      <div class="kChart">
+        <div echarts class="kChart" [options]="_lineChartData[_selectedMetrics]" *ngIf="_chartType==='line'"></div>
```

Comment:
@diamond-darrell - we can remove the _chartType logic as there is only line chart in this report